experimental/add_to_app/ios_using_prebuilt_module/IOSUsingPrebuiltModuleUITests/IOSUsingPrebuiltModuleUITests.swift

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import XCTest
```

Comment:
Should we delete the test target since it's just the boilerplate?  The test target is missing the framework search paths it would need to test Flutter bits.

Reply:
I've deleted the tests for now. We don't have a great test strategy for add-to-app samples in general right now, which is something I intend to fix over the holidays.